queries.sql

```diff
@@ -1,3 +1,18 @@
 /*Queries that provide answers to the questions from all projects.*/
+-- SELECT * from animals WHERE name = 'Luna';
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT name FROM animals WHERE name LIKE '%mon';
+
+SELECT * FROM animals WHERE EXTRACT(year FROM date_of_birth)BETWEEN 2016 AND 2019;
```

Comment:
- [ ] I would suggest you go through all the names of the animals and list the ones that were born between 2016 and 2019.
`SELECT name FROM animals WHERE date_of_birth >= '2016/01/01' AND date_of_birth <= '2019/31/12';`

Reply:
Done